python/source/org/libreoffice/ide/eclipse/python/PythonBuilder.java

```diff
@@ -205,57 +173,7 @@ private void runJavamaker(String firstModule, String oooTypesArgs,
     @Override
     public String[] getBuildEnv(IUnoidlProject pUnoProject) {
 
-        IProject project = ResourcesPlugin.getWorkspace().getRoot().getProject(pUnoProject.getName());
-
-        String[] env = new String[2];
-
-        // compute the classpath for the project's OOo instance
-        String classpath = "CLASSPATH="; //$NON-NLS-1$
-        String sep = System.getProperty("path.separator"); //$NON-NLS-1$
-
-        File javaHomeFile = null;
-
-        // Compute the classpath for the project dependencies
-        IJavaProject javaProject = JavaCore.create(project);
-        if (javaProject != null) {
-            try {
-                IClasspathEntry[] cpEntry = javaProject.getResolvedClasspath(true);
-                for (int i = 0; i < cpEntry.length; i++) {
-                    IClasspathEntry entry = cpEntry[i];
-
-                    // Transform into the correct path for the entry.
-                    if (entry.getEntryKind() != IClasspathEntry.CPE_SOURCE) {
-                        classpath += entry.getPath().toOSString();
-                    }
-                    if (i < cpEntry.length - 1) {
-                        classpath += sep;
-                    }
-                }
-
-                IVMInstall vmInstall = JavaRuntime.getVMInstall(javaProject);
-                javaHomeFile = vmInstall.getInstallLocation();
-
-            } catch (JavaModelException e) {
-                PluginLogger.error(
-                    Messages.getString("Language.GetClasspathError"), e); //$NON-NLS-1$
-            } catch (CoreException e) {
-                // TODO log a problem to find the JVM associated to the project
-            }
-        }
-
-        env[0] = classpath;
-        if (javaHomeFile != null) {
-            String libs = ""; //$NON-NLS-1$
-            String filesep = System.getProperty("file.separator"); //$NON-NLS-1$
-            try {
-                String arch = System.getProperty("os.arch"); //$NON-NLS-1$
-                libs = javaHomeFile.getCanonicalPath() + filesep + "lib" + filesep + arch; //$NON-NLS-1$
-            } catch (IOException e) {
-            }
-            env[1] = "LD_LIBRARY_PATH=" + libs; //$NON-NLS-1$
-        }
-
-        return env;
+        return new String[0];
```

Comment:
The jar files related to libreoffice and java are not required any more. Therefore removing them, so that they are uselessly not added into the list of environment variables when we are trying to run it as LibreOffice Extension directly.